vita3k/gui/src/information_bar.cpp

```diff
@@ -356,7 +356,7 @@ static void draw_notice_info(GuiState &gui, HostState &host) {
         if (gui.theme_information_bar_notice.find("no") != gui.theme_information_bar_notice.end())
             ImGui::GetForegroundDrawList()->AddImage(gui.theme_information_bar_notice["no"], NOTICE_POS, ImVec2(NOTICE_POS.x + NOTICE_SIZE.x, NOTICE_SIZE.y));
         else
-            ImGui::GetForegroundDrawList()->AddRectFilled(ImVec2(display_size.x - (70.f * SCAL.x), (-24.f * SCAL.y)), ImVec2(display_size.x + (14.f * SCAL.y), 60.f * SCAL.y), IM_COL32(62.f, 98.f, 160.f, 255.f), 75.f * SCAL.x, ImDrawCornerFlags_All);
+            ImGui::GetForegroundDrawList()->AddCircleFilled(ImVec2(display_size.x - (28.f * SCAL.x), (18.f * SCAL.x)), 44.0f * SCAL.x, IM_COL32(62.f, 98.f, 160.f, 255.f));
```

Comment:
i have try is great :D, add same on 353 line just before, is notif new, size is little more big try calculate for get same position :)